### DISCUSSION TITLE
ArtifactUpload>DataPipeline are pointed to Azure container for GCPdeployment

### DISCUSSION THREAD

**Discussion by tsprasath** [Sunbird-Ed/Community#272] [Installation] [answered]
Hi Team

CSP: GCP

Try to build and artifact upload jobs. The below jobs are points to Azure container instead of GCP storage

- AdhocScripts - release-4.5.1_RC1
- AnalyticsCore - release-4.10.5_RC1
- CoreDataProducts - release-4.10.5_RC1
- EdDataProducts - release-4.10.5_RC3
- ETLJobs - release-4.10.5_RC2

Error:
`TASK [artifacts-upload-azure : Ensure azure blob storage container exists] *****
fatal: [localhost]: FAILED! => {
    "changed": true,
    "cmd": [
        "az",
        "storage",
        "container",
        "create",
        "--name",
        "artifacts"
    ],
    "delta": "0:01:23.713885",
    "end": "2022-11-24 19:25:49.472676",
    "rc": 1,
    "start": "2022-11-24 19:24:25.758791"
}

STDERR:

ERROR: <urllib3.connection.HTTPSConnection object at 0x7f841d7c0df0>: Failed to establish a new connection: [Errno -2] Name or service not known

MSG:

non-zero return code`

**Comment by beepdot** [comment]
Please note, 5.1.0 release branch is still in development phase due to which you may get many such issues.
What is the value defined for https://github.com/project-sunbird/sunbird-devops/blob/release-5.1.0/private_repo/ansible/inventory/dev/Core/common.yml#L4 ?

**Reply by tsprasath** [reply]
@keshavprasadms below is the value used

`cloud_service_provider: "gcloud"`

**Reply by beepdot** [reply]
Same issue as https://github.com/orgs/Sunbird-Ed/discussions/273
Answers will be posted in the linked discussion.

**Comment by tsprasath** [comment, marked as answer]
@keshavprasadms not required as part for Sunbird ED setup.Please close this thread
